File: RealEstate/RealEstate.Models/Dtos/EstateDtos/EstateDetailsDto.cs
using RealEstate.Models.Enums;

namespace RealEstate.Models.Dtos.EstateDtos
{
    public class EstateDetailsDto
    {
        public int Id { get; set; }

        public int EstateBrokerId { get; set; }

        public string EstateBrokerUsername { get; set; }

        public EstateType Type { get; set; }

        public decimal Price { get; set; }

        public int Size { get; set; }

        public int SettlementId { get; set; }

        public string SettlementName { get; set; }

        public int? Floor { get; set; }
    }
}

File: real-estate/src/estate/dtos/estate-details-dto.ts
import { EstateType } from "../enums/estate-type";

export class EstateDetailsDto {
  id: number;
  estateBrokerId: number;
  estateBrokerUsername: string;
  type: EstateType;
  price: number;
  size: number;
  settlementId: number;
  settlementName: string;
  floor?: number;
}
